scripts/django_batch_loader.py

```diff
@@ -0,0 +1,160 @@
+            'driverIncidentDetails': dict(),
+            'driverPerson': [],
+            'driverVehicle': []
+        },
+        'schema': str(schema_id),
+        'occurred_from': 'None',
+        'occurred_to': 'None',
+        'geom': 'POINT (0 0)'
+    }
+    data = obj['data']
+
+    # Skip if no date
+    if not record['record_date']:
+        return
+
+    for key, value in details_mapping.iteritems():
+        if key in record:
+            data['driverIncidentDetails'][value] = record[key]
+    data['driverIncidentDetails']['Location Approximate'] = 'Yes'
+    data['driverIncidentDetails']['Reporting Agency'] = 'MMDA Road Safety Unit'
+
+    # Add in the _localId field; they're not used here but the schema requires them
+    def _add_local_id(dictionary):
+        dictionary['_localId'] = str(uuid.uuid4())
+
+    _add_local_id(data['driverIncidentDetails'])
+
+    # Set the occurred_from/to fields
+    occurred_date = parser.parse(record['record_date'])
+    occurred_date = pytz.timezone('Asia/Manila').localize(occurred_date)
+    obj['occurred_from'] = occurred_date.isoformat()
+    obj['occurred_to'] = occurred_date.isoformat()
+
+    obj['geom'] = 'POINT ({} {})'.format(float(record['lon']), float(record['lat']))
+
+    # Add vehicles
+    mode = [x.strip() for x in record['mode'].split(',')]
+    modes_with_count = [x.split(' ') for x in mode]
+    try:
+        modes_with_count = [[int(float(x[0])), x[1]] for x in modes_with_count]
+    except ValueError:  # If count data is bad just return what we have
+        return obj
+    modes_with_count = [x for x in modes_with_count if x[0]]
+    mode_map = {
+        'car': 'Car',
+        'bus': 'Bus',
+        'jeepney': 'Jeepney',
+        'truck': 'Truck (Unknown)',
+        'van': 'Van',
+        'motorcycle': 'Motorcycle',
+        'tricycle': 'Tricycle',
+        'bike_or_pedicab': 'Bicyle',
+        'train': 'Heavy Equipment',
+        'kalesa': 'Horse-Driven Carriage (Tartanilla)',
+        'pushcart': 'Push-Cart'
+    }
+
+    if len(modes_with_count):
+        for mode_with_count in modes_with_count:
+            count = mode_with_count[0]
+            mode = mode_with_count[1]
+            for i in range(count):
+                obj['data']['driverVehicle'].append({
+                    'Vehicle type': mode_map[mode],
+                    '_localId': str(uuid.uuid4())
+                })
+
+    return obj
+
+
+def load(obj, api, schema, headers=None):
+    """Load a transformed object into the data store"""
+    if not obj:
+        return
+
+    if headers is None:
+        headers = {}
+
+    Record.objects.create(
+        occurred_from=obj['occurred_from'],
+        occurred_to=obj['occurred_to'],
+        geom=obj['geom'],
+        schema=schema,
+        data=obj['data']
+    )
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Load incidents data (v3)')
+    parser.add_argument('incidents_csv_dir', help='Path to directory containing incidents CSVs')
```

Comment:
I think this is actually ignored, but that seems like bad UX, so maybe it'd be better to move this to a parameter of main(), substitute it for `batch` on line 145, and update the sample docs.

Reply:
I don't think I understand what you mean by substitute it for batch. I can just completely remove the parser, right? And this being the Django load, there wouldn't be any update needed to the docs?

Reply:
Ohh, sorry, I missed that you added instructions to copy CSV files into the `batch` folder. Yeah, this can just be deleted then. 👍